rules/docker-fpm-frr.mk

```diff
@@ -24,7 +24,6 @@ SONIC_STRETCH_DBG_DOCKERS += $(DOCKER_FPM_FRR_DBG)
 $(DOCKER_FPM_FRR)_CONTAINER_NAME = bgp
 $(DOCKER_FPM_FRR)_RUN_OPT += --privileged -t
 $(DOCKER_FPM_FRR)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
-$(DOCKER_FPM_FRR)_RUN_OPT += -v /etc/sonic/frr:/etc/frr:rw
```

Comment:
This line is needed for persist config. Why remove?

Reply:
This line is added in docker_img_ctl.j2, so that the instance specific frr directory is mounted. For ex: /etc/sonic/frr0 for namespace(or ASIC)0.